.github/CONTRIBUTING.md

```diff
@@ -11,65 +10,74 @@ with git](https://help.github.com/en/github/using-git) and [here’s some help t
 * Change into the directory you’d like
 * Clone your forked repo
 
-        git clone https://github.com/[yourgithubname]/three.js.git
+      git clone https://github.com/[yourgithubname]/three.js.git
+
 * Go into the three.js directory.
         
-        cd ./three.js
+      cd ./three.js
+
 * Install the dependencies
 
-        npm install
+      npm install
 
-#### Next Steps
+## Next Steps
 
 As per the npm standard, ‘start’ is the place to begin the package.
 
     npm start
 
-This script will start a local server similar to three.js, but instead will be hosted on your local
-machine. Browse to http://localhost:8080/ to check it out. It also automatically creates the
-‘build/three.js’ and ‘build/three.module.js’ scripts anytime there is a change within your three.js
-directory.
+This script will start a local server similar to three.js, but instead will be hosted on your local machine. Browse to http://localhost:8080/ to check it out. It also automatically creates the ‘build/three.js’ and ‘build/three.module.js’ scripts anytime there is a change within your three.js directory.
```

Comment:
> This script will start a local server similar to three.js

What does this mean? Does it mean similar to https://threejs.org?

Reply:
yup. oh, right, wording. I'll change it to:

local server similar to [threejs.org](https://threejs.org/),